src/test/connection_snippets.rs

```diff
@@ -0,0 +1,20 @@
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+
+use crate::error::Result;
+use crate as mongodb;
+
+#[cfg(feature = "tokio-runtime")]
+async fn connect() -> Result<()> {
```

Comment:
This will fail if we actually run it since the URI has placeholders

Reply:
I think this might cause some lint warnings due to stuff not being used; could you take a look at the CI output and annotate the function accordingly to suppress the warnings?

Reply:
looks like it was just formatting, fixed